doc/classes/AnimationTreePlayer.xml

```diff
@@ -610,6 +619,16 @@
 	</methods>
 	<members>
 		<member name="playback_process_mode" type="int" setter="set_animation_process_mode" getter="get_animation_process_mode" enum="AnimationTreePlayer.AnimationProcessMode">
+			The process notification in which to update animations. Default value: [enum ANIMATION_PROCESS_IDLE].
```

Comment:
Maybe:
The thread in which to update animations. Default value: [enum ANIMATION_PROCESS_IDLE].

Also:
I would also document the enums themselves, with the text from AnimationPlayer:
ANIMATION_PROCESS_PHYSICS = 0 Process animation during the physics process. This is especially useful when animating physics bodies.
ANIMATION_PROCESS_IDLE = 1 Process animation during the idle process.